src/js/components/mapWidgets/widgetContent/shareContent.tsx

```diff
@@ -49,18 +54,26 @@ const ShareContent: FunctionComponent = () => {
     // * because it doesn't support localhost URLs
   };
 
+  const returnURL = (): string => {
+    const { latitude, longitude, zoom } = mapController.setMapviewCoordinates();
```

Comment:
can we rename method to be called `getMapviewCoordinates` ? I think GET instead of SET would indicate better what are we trying to achieve here. Thoughts?